packages/microfrontend-sdk/src/MicrofrontendIntegrator.test.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { JSDOM } from 'jsdom'
```

Comment:
Instead of this, it should not be possible with jest to use `testEnvironment` set to [jsdom](https://jestjs.io/docs/configuration#testenvironment-string)?

Reply:
Moreover, it is jsdom needed?

Reply:
Simulated node creation is not so useful for the tests. i decide to remove it